freebsd-kvm/buildkite-worker/setup_scripts/install-buildkite-agent.sh

```diff
@@ -91,7 +91,8 @@ buildkite_start() {
         \${buildkite_env} \
         HOME=\$(pw usershow \${buildkite_account} | cut -d: -f9) \
         BUILDKITE_AGENT_TOKEN=\${buildkite_token} \
-        /usr/local/bin/buildkite-agent start --config \${buildkite_config}"
+        /usr/local/bin/buildkite-agent start --config \${buildkite_config} \
+        || true"
```

Comment:
```suggestion
        /usr/local/bin/buildkite-agent start --config \${buildkite_config}" \
        || true
```
Per discussion in Slack it sounds like this is the change Elliot made